src/js/view/configurations/configureStore.test.js

```diff
@@ -0,0 +1,23 @@
+import { configureStore } from './configureStore';
+
+it('should saveStateToDisk', () => {
+  const fakeStore = {
+    getState: () => {
+      console.log('getState invoked');
```

Comment:
Is this console log necessary?

Reply:
Hela funktionen kan me fördel ersättas med en https://jestjs.io/docs/en/mock-functions